src/Server.php

```diff
@@ -37,13 +35,13 @@ public function addFunction($function, $namespace = '');
      * Namespacing is primarily for xmlrpc, but could be used for other
      * implementations as well.
      *
-     * @param  mixed $class Class name or object instance to examine and attach
-     *                      to the server.
+     * @param  string|object $class Class name or object instance to examine and attach
+     *                              to the server.
```

Comment:
This looks related to https://github.com/laminas/laminas-coding-standard/issues/13
This is alignment I prefer too see:
```suggestion
     *     to the server.
```

Reply:
I've added this alignment as it was something we have in this library in most of the places.
It will be changed in next major when it will be properly defined in our CS rules.